java/src/jmri/jmrix/srcp/SRCPTurnoutManager.java

```diff
@@ -12,23 +13,34 @@
  */
 public class SRCPTurnoutManager extends jmri.managers.AbstractTurnoutManager {
 
-    int _bus = 0;
+    //private int _bus = 0; // not used, if needed, get bus via {@link SRCPBusConnectionMemo#memo.getBus()}
```

Comment:
If this is no longer required, just remove it, there isn’t a reason to keep a private variable like this around.

Reply:
done